docs/source/en/using-diffusers/loading_adapters.md

```diff
@@ -307,3 +307,74 @@ prompt = "a house by william eggleston, sunrays, beautiful, sunlight, sunrays, b
 image = pipeline(prompt=prompt).images[0]
 image
 ```
+
+### IP-Adapter 
+
+[IP-Adapter](https://ip-adapter.github.io/) is an effective and lightweight adapter to achieve image prompt capability for the pre-trained text-to-image diffusion models. It is now available to use with most of our Stable Diffusion and Stable Diffusion XL pipelines. You can also use the IP-Adapter with other custom models fine-tuned from the same base model, as well as ControlNet and T2I adapters. Moreover, the image prompt can also work well with the text prompt to accomplish multimodal image generation.
+
+Let's look at an example where we use IP-Adapter with the Stable Diffusion text-to-image pipeline.
+
+``` py
+from diffusers import AutoPipelineForText2Image
+import torch
+from diffusers.utils import load_image
+
+pipeline = AutoPipelineForText2Image.from_pretrained("runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16).to("cuda")
```

Comment:
This doesn't load the `image_encoder`.

Reply:
oh yeah we are not going to add it to the sd folder?

Reply:
we can add it right? it is pretrained

Reply:
Discussed via Slack. Since the original checkpoints don't ship the `image_encoder` officially, it won't make sense for us to add it now I think. So, it's better to be explicit here and it doesn't hurt either as it's just a one-liner to initialize the `image_encoder`.

Reply:
No we won't be able to add this to the SD folder. We'll have to always pass it manually